pkg/component/etcd/service/values_helper.go

```diff
@@ -47,6 +47,7 @@ func GenerateValues(etcd *druidv1alpha1.Etcd) Values {
 		EtcdName:                 etcd.Name,
 		EtcdUID:                  etcd.UID,
 		Labels:                   etcd.Spec.Labels,
+		Selectors:                etcd.Spec.Selector.MatchLabels,
```

Comment:
etcd CRD has `etcd.clientService.labels`? This section of the etcd CRD is not used today. Any reason why? Also is there a need for users to set additional labels for Services (client and peer)?